src/components/SuggestedSearches/SuggestedSearches.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './SuggestedSearches.styles.css';
+import GetSuggestedSearches from './GetSuggestedSearches.component';
+// import { useAuth } from '../../providers/Auth';
+
+export default function Sugestedsearches() {
+  const randomSuggestedSearches = GetSuggestedSearches();
+  // const { authenticated, userInfo } = useAuth();
+
+  // console.log(authenticated);
+  // console.log(userInfo);
+  // {authenticated ? `Welcome back ${userInfo.name}.` : ''}
+  return (
+    <div className="suggested_searches">
+      <p>Try this searches:</p>
+      <div className="suggested_searches_list"> {randomSuggestedSearches} </div>
```

Comment:
We could also try to use the component itself here
```js
<div className="suggested_searches_list">
    <GetSuggestedSearches />
</div>
```

Reply:
At first I was doing it that way, but later I change my mind to return only the list of words and put the text `try this searches:` outside the component.